lua/nvim-web-devicons.lua

```diff
@@ -1656,6 +1656,12 @@ local icons_by_file_extension = {
     cterm_color = "106",
     name = "Liquid",
   },
+  ["Dockerfile"] = {
```

Comment:
We don't use specific cases for the filenames or extensions as `get_icon` is not case sensitive.

Reply:
This is interesting. Currently i have the following configuration for `nvim-web-devicons` (notice the uppercase "Dockerfile"):

```lua
override_by_extension = {
    ["Dockerfile"] = {
        icon = "󰡨",
        color = "#458ee6",
        cterm_color = "68",
        name = "Dockerfile",
    },
},
```

if i chance it to be lowercased "dockerfile" the below example with `migrations.Dockerfile` doesn't get an icon